marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-match-merge-reader.json

```diff
@@ -0,0 +1,4 @@
+{
+  "role-name": "data-hub-match-merge-reader",
+  "description": "Permits reading a matching configuration"
```

Comment:
Is the assumption that this role will allow access to either match or merge configurations? If so, the description should state that.

Reply:
Thanks @rjrudin . I have changed the description.